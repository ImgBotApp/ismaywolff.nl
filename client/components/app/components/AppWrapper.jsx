import styled from 'styled-components'

const AppWrapper = styled.div`
  margin: 0 auto;
  padding: 1rem;
  max-width: 30rem;
`

export default AppWrapper
