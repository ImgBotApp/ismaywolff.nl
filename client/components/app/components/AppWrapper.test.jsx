import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import AppWrapper from './AppWrapper'

describe('<AppWrapper />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AppWrapper />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
