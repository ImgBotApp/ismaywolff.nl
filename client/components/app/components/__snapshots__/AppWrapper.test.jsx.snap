// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppWrapper /> renders correctly 1`] = `
<div
  className="cesyxe"
/>
`;
